README.md

```diff
@@ -1,2 +1,136 @@
 # rtsan-standalone-rs
-Rust crate repo for rtsan standalone
+
+This is a wrapper for the standalone version of RealtimeSanitizer (RTSan) to
+detect real-time violations in Rust applications.
```

Comment:
In this intro section, I would suggest linking to both the official clang docs and our helper repo.